index.html

```diff
@@ -308,20 +308,155 @@
 
     <section class="informative">
       <h2>Introduction</h2>
+
+      <p>
+The use of shared secrets for [=authentication=] and [=authorization=], such as
+the use of passwords, has resulted in a variety of security failures over the
+past several decades. To address these security failures, systems can upgrade to
+the use of
+<a href="https://en.wikipedia.org/wiki/Public-key_cryptography">
+asymmetric cryptography</a>, which uses digital signatures that are far more
+difficult to compromise. However, one shortcoming of digital signatures is
+the difficulty in disseminating the information, such as public cryptographic
+keys, to those that would need to verify the security of the digital signature.
+      </p>
+
       <p>
 A [=controller document=] is a set of data that specifies one or more
-relationships between a [=controller=] and a set of data, such as a set of
-public cryptographic keys. The [=controller document=] contains [=verification
-relationships=] that explicitly permit the use of certain [=verification
-methods=] for specific purposes.
+relationships between an identifier that is controlled by a [=controller=] and a
```

Comment:
There's another PR out there #102 that has text that conflicts with this (I think).

Reply:
Yes, this text will just re-use what's in #102 eventually.